src/openrct2/world/Map.cpp

```diff
@@ -1096,6 +1099,8 @@ void map_reorganise_elements()
         for (int32_t x = 0; x < MAXIMUM_MAP_SIZE_TECHNICAL; x++)
         {
             TileElement* startElement = map_get_first_element_at(x, y);
+            if (startElement == nullptr)
+                return;
```

Comment:
This would leak memory. Use `continue;`. If you ever hit this i suspect leaking memory would be the least of your worries though.